nginx-fem-staging-redirects.conf

```diff
@@ -0,0 +1,23 @@
+set $fe_project_uri "https://fe-project.preview.zooniverse.org";
+set $fe_content_pages_uri "https://fe-content-pages.preview.zooniverse.org";
+
+# FEM projects app
+location ~* ^/projects/.*$ {
```

Comment:
this means paths like `/projects/camallen/testing-project/talk` won't work via PFE fallback at all. If we want the frontend site to preview as our production systems will then we will need rules that are more specific and match the prod routing we have

I'm happy to produce these location blocks and push to this branch if you want.

Reply:
Yeah, these are copied straight from the existing FrontDoor setup and need refining so that we can stage projects with working pages for Talk etc.

Reply:
I've tightened this up so that the rule only captures URLs for project home, about and classify pages.